chat/src/main/java/com/stfalcon/chatkit/messages/SkygearChatMessageHolders.kt

```diff
@@ -0,0 +1,35 @@
+package com.stfalcon.chatkit.messages
+
+import android.util.SparseArray
+import android.view.View
+import android.widget.ImageButton
+import com.stfalcon.chatkit.commons.ImageLoader
+import com.stfalcon.chatkit.commons.ViewHolder
+import com.stfalcon.chatkit.commons.models.IMessage
+import com.stfalcon.chatkit.utils.DateFormatter
+import io.skygear.plugins.chat.ui.model.VoiceMessage
+import io.skygear.plugins.chat.R
+
+class SkygearChatMessageHolders : MessageHolders {
+    var voiceMessageOnClickListener: VoiceMessageOnClickListener?
+    constructor(voiceMessageOnClickListener: VoiceMessageOnClickListener?): super() {
+        this.voiceMessageOnClickListener = voiceMessageOnClickListener
+    }
+
+    override fun bind(holder: ViewHolder<*>?,
```

Comment:
Let write down some note on what are we overriding here since it is not intuitive.